pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -82,7 +83,7 @@ export class JournalOutput {
                     count > 1
                         ? <div className="cockpit-log-service-container" role="cell">
                             <div className="cockpit-log-service-reduced" role="cell">{ident}</div>
-                            <span className="badge" role="cell">{count}&#160;<i className="fa fa-caret-right" /></span>
+                            <Badge isRead key={count} role="cell">{count}&#160;<AngleRightIcon /></Badge>
```

Comment:
Do we need that `>`?
I never understood what it was trying to say.
For example we have the same logic of "squash same reports" in selinux and we don't show that icon there:
![counter](https://user-images.githubusercontent.com/12330670/110685469-98ba4880-81de-11eb-9c55-9136db03d1df.png)

Reply:
The difference between selinux and logs panel is that in the logs that badge is actually redirecting to the logs details when clicked on. I agree that's not obvious at all though that this arrow suggests that the badge contains a link.

Reply:
To be clear, it's not the badge that it's redirecting, the whole line is having the onClick handlers.

Reply:
Heh, we commented together. Shall I repost then removing the arrows and making the onClick be on the text?

Reply:
@garrett WDYT
- is that arrow necessary? (This predates me, so I am not sure, if there has been some reason behind this)
- Would it be fine, if only text would redirect and not the whole line? Now on hover I think just the text changes color and cursor.

Reply:
@marusak How about I post a followup PR on this? I have the code change locally in fact, but since this is fine from @garrett and the tests seem happy I would rather merge and introduce the design change in a seperate PR.

Reply:
Agree! Lets not block on this, especially since this blocks another PR.

Reply:
I see an issue with making the message the only clickable part of a row:

* If we make it a link or a button it will become blue by default (probably we don't want that)
* If we don't make it link or a button it will not be blue and the users will not have any idea where to click on the row to get redirected. Now indeed we only underline the message on hover, but at least clicking anywhere in the row will redirect.

Reply:
I don't think the arrow is necessary. 

It doesn't do anything. From my understanding, the number is just the count of the number of the times it is repeated. I never understood why it had an arrow. (It predates me working on Cockpit too.)

If we all agree it isn't necessary, let's just drop it?

Reply:
Agree! We will do this as follow up though as we want to move this forward.
@KKoukiou will you send PR for that?